geode-experimental-driver/src/test/java/org/apache/geode/experimental/driver/DriverConnectionTest.java

```diff
@@ -96,6 +94,18 @@ public void driverCanConnectWhenThereAreServers() throws Exception {
     server.setPort(0);
     server.start();
     driver = new DriverFactory().addLocator("localhost", locatorPort).create();
+    assertTrue(driver.isConnected());
+  }
+
+  @Test
+  public void driverReportsItIsDisconnected() throws Exception {
+    CacheServer server = cache.addCacheServer();
+    server.setPort(0);
+    server.start();
+    driver = new DriverFactory().addLocator("localhost", locatorPort).create();
+    driver.close();
```

Comment:
What about `assertTrue(driver.isConnected());` _before_ `driver.close()`?

Reply:
Good idea - I'll add that to the test

Reply:
Oh, wait!  I already have that check in driverCanConnectWhenThereAreServers.  Do you think I also need it in this test?